user-operator/src/test/java/io/strimzi/operator/user/model/KafkaUserModelTest.java

```diff
@@ -233,8 +232,10 @@ public void testGenerateSecretGeneratesPasswordWhenNoUserSecretExists()    {
                         .withKubernetesManagedBy(KafkaUserModel.KAFKA_USER_OPERATOR_NAME)
                         .toMap()));
 
-        assertThat(generatedSecret.getData().keySet(), is(singleton(KafkaUserModel.KEY_PASSWORD)));
+        assertThat(generatedSecret.getData().keySet(), is(set(KafkaUserModel.KEY_PASSWORD, KafkaUserModel.KEY_JAAS_CONFIG)));
         assertThat(new String(Base64.getDecoder().decode(generatedSecret.getData().get(KafkaUserModel.KEY_PASSWORD))), is("aaaaaaaaaa"));
+        assertThat(new String(generatedSecret.getData().get(KafkaUserModel.KEY_JAAS_CONFIG)), is("org.apache.kafka.common.security.scram.ScramLoginModule required username=\"" + ResourceUtils.NAME + "\" password=\"YWFhYWFhYWFhYQ==\";"));
```

Comment:
I'm a bit confused by this ... why does it not need base64 decoding? And why is the password here not `the same as it is above?